templates/master/00-master/baremetal/files/baremetal-chrony.yaml

```diff
@@ -0,0 +1,48 @@
+filesystem: "root"
+mode: 0644
+path: "/etc/chrony.conf"
+contents:
+  inline: |
+    #
+    # Configure an NTP server accessible on the redhat network.
```

Comment:
This isn't suitable for upstreaming.  See also https://github.com/openshift/machine-config-operator/issues/629

Reply:
I'll remove this.